WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -31,22 +31,31 @@
 from DIRAC.StorageManagementSystem.Client.StorageManagerClient import StorageManagerClient
 
 # This is a global instance of the JobDB class
-gJobDB = False
-gJobLoggingDB = False
-gtaskQueueDB = False
-gPilotAgentsDB = False
-gPilotsLoggingDB = False
+gJobDB = None
+gJobLoggingDB = None
+gtaskQueueDB = None
+gPilotAgentsDB = None
+gPilotsLoggingDB = None
+enablePilotsLogging = None
 
 MAX_PARAMETRIC_JOBS = 20
 
 def initializeJobManagerHandler( serviceInfo ):
 
-  global gJobDB, gJobLoggingDB, gtaskQueueDB
+  global gJobDB, gJobLoggingDB, gtaskQueueDB, enablePilotsLogging
   gJobDB = JobDB()
   gJobLoggingDB = JobLoggingDB()
   gtaskQueueDB = TaskQueueDB()
-  gPilotAgentsDB = PilotAgentsDB()
-  gPilotsLoggingDB = PilotsLoggingDB()
+
+  result = gConfig.getValue('/Services/PilotsLogging/Enable')
+  if not result['OK']:
+    return result
+  if result['Value'] == 'Yes':
+    enablePilotsLogging = True
```

Comment:
`gConfig.getValue` does not return `S_OK/S_ERROR ` structure.

You could also call it like:
```
enablePilotsLogging = gConfig.getValue('/Services/PilotsLogging/Enable', 'False').lower() in ('yes', 'true')
```